.travis.yml

```diff
@@ -3,16 +3,18 @@ sudo: required
 
 # Install Bazel and set up GOPATH.
 before_install:
-- go get -u github.com/alecthomas/gometalinter
-- wget https://github.com/bazelbuild/bazel/releases/download/0.4.4/bazel_0.4.4-linux-x86_64.deb
-- sudo dpkg -i bazel_0.4.4-linux-x86_64.deb
 - mkdir -p $GOPATH/src/k8s.io
 - mv $TRAVIS_BUILD_DIR $GOPATH/src/k8s.io
 - cd $GOPATH/src/k8s.io/repo-infra
+- echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+- curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
+- sudo apt-get update
 
 install:
-  - gometalinter --install
+  - sudo apt-get install bazel
```

Comment:
Most of us probably have bazel auto-updating anyway, so I'm not sure how much pinning to a version gains us.

The debs on github don't expire (we'd been installing https://github.com/bazelbuild/bazel/releases/download/0.4.4/bazel_0.4.4-linux-x86_64.deb for a while), but those debs started failing for reasons I don't understand:
```
0.11s$ sudo dpkg -i bazel_0.4.4-linux-x86_64.deb
Selecting previously unselected package bazel.
dpkg: regarding bazel_0.4.4-linux-x86_64.deb containing bazel:
 bazel conflicts with openjdk-9-jdk
  oracle-java9-installer provides openjdk-9-jdk and is present and installed.
dpkg: error processing archive bazel_0.4.4-linux-x86_64.deb (--install):
 conflicting packages - not installing bazel
Errors were encountered while processing:
 bazel_0.4.4-linux-x86_64.deb
```

0.5.2 has the same issue.

IMO the real fix is that we should just use prow.